winton_kafka_streams/errors/task_migrated_error.py

```diff
@@ -0,0 +1,9 @@
+"""
+Indicates that a task got migrated to another thread.
+Thus, the task raising this exception can be cleaned up and closed as "zombie".
+"""
```

Comment:
Could put this docstring into the class below, then it'll show up nicely in future docs etc.

http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

```python
class TaskMigratedError(KafkaStreamsError): 
    """
    Indicates that a task got migrated to another thread.
    Thus, the task raising this exception can be cleaned up and closed as "zombie".
    """
    pass
```

Reply:
Done.